### DISCUSSION TITLE
Suggestions for #hiring-job-board offers

### DISCUSSION THREAD

**Discussion by smoya** [bcneng/candebot#54] [Ideas]
Suggest changes about the format on the jobs posted in #hiring-job#board channel.

**Comment by mobregozo** [comment]
One suggestion I have already shared on Slack is about the need for having a field that determines the type of contract the company is offering.
The reason behind this is the difference between being a full-time employee and a contractor/freelancer, in terms of money expectations, taxes, holidays, pension plan, equity, duration of the contract, among others.  

**Suggestion**:
Having a "Type of Job" field with different options:
-  `employee` or `full-time`
-  `contractor` or `freelance`

I have checked some of the job posting portals that I normally use and that seems to be the most common pattern. 
We can use this as a starting point and iterate later on with other options based on the feedback.

**Reply by joanviladomat** [reply]
The only thing is that sometimes it can be a full-time job but the employer is only offering a freelance contract because they don't have a legal company in Spain. So, sometimes can be full-time and freelance at the same time.

**Reply by mobregozo** [reply]
True, well I guess then we can combine both and make it `full-time employee`, or something similar to avoid the edge case you have commented on.

**Comment by joanviladomat** [comment]
In the dropdown "Published by", right now there are 3 options: Employer, Agency and Referral. Not sure if necessary to have the option "Referral". In my opinion, it doesn't add a lot of value and at the same time generates complexity. After discussing, several people understood "Referral" in different ways. 

Maybe it's better to have just "Employer" and "Agency". Then if it's posted by an employee of the company, it's clear that you need to select "Employer" either if you are an internal recruiter or any other employee.

**Comment by joanviladomat** [comment]
In the dropdown "Published by", there is the option "Agency". Not sure if it's the best word because for some developers "Agency" can have a negative reputation. Also, there are a lot of different types of channels to attract tech talent.  Some of them: "Recruitment Agency", "Job Board", "Freelance Recruiter", "Tech Jobs Marketplace", "Job Search App"...

I would go for a more general word to fit all the options. Some ideas:
- Partner (my favourite)
- Provider
- External
- External Source
- External Channel
...

Any other ideas?

**Comment by Gery-Wallapop** [comment]
Hi! 
when we post a job we don't have the option of a Hybrid model, the one of Barcelona/remote can be misinterpreted ?

![models](https://user-images.githubusercontent.com/106662466/200269143-fb675c71-82a6-43ac-b3fc-c11929a19d5a.png)

**Reply by smoya** [reply]
I agree those should be updated. The same for the one that mentions COVID (should be removed). 

@Gery-Wallapop Would you mind creating a PR with such a change so we can discuss right in the code? Here the lines that should be changed https://github.com/bcneng/candebot/blob/main/bot/interact.go#L289-L305

**Comment by afminguela** [comment]
# Post Public Job Offers

hi, will be great a way to  post public job offers and distinguish it from normal ones: 
The diferencies are: no referral options, process explained on official webs and BOE/BOGC/Ayuntamientos… and Study syllabus.

**Comment by RafaelAybar** [comment]
Hi, I consider that would be adding a checkbox or similar that specifies if the remote work is hybrid. In addition, we can add a numeric field for "the office days".
